wallet/wallet.go

```diff
@@ -3907,7 +3907,7 @@ func decodeStakePoolColdExtKey(encStr string, params *chaincfg.Params) (map[stri
 
 // Open loads an already-created wallet from the passed database and namespaces.
 func Open(db walletdb.DB, pubPass []byte, votingEnabled bool, addressReuse bool,
-	ticketAddress string, poolAddress string, poolFees float64, ticketFee float64,
+	ticketAddress *dcrutil.Address, poolAddress *dcrutil.Address, poolFees float64, ticketFee float64,
```

Comment:
dcrutil.Address is an interface and should be passed by value here, not by pointer.